src/main/java/org/jabref/logic/preferences/JabRefCliPreferences.java

```diff
@@ -277,6 +277,9 @@ public class JabRefCliPreferences implements CliPreferences {
     public static final String OO_CURRENT_STYLE = "ooCurrentStyle";
     public static final String OO_ALWAYS_ADD_CITED_ON_PAGES = "ooAlwaysAddCitedOnPages";
 
+    public static final String CSL_BIBLIOGRAPHY_TITLE = "bibliographyTitle";
```

Comment:
Repeating this comment again - I am not sure again why you have marked it resolved. (https://github.com/JabRef/jabref/pull/12784#discussion_r2010525121).

```suggestion
    public static final String CSL_BIBLIOGRAPHY_TITLE = "cslBibliographyTitle";
```